src/ngraph/runtime/gpu/gpu_runtime_context.hpp

```diff
@@ -40,13 +40,19 @@ namespace ngraph
                 gpu::primitive* const* gpu_primitives;
                 const gpu::memory_primitive* gpu_memory_primitives;
                 CudaFunctionPool* compiled_kernel_pool;
+                StopWatchPool* stopwatch_pool = nullptr;
```

Comment:
The stopwatch pool is now in the runtime context, but it is being called from the JIT runtime which will still relies on the virtual function table to correctly dispatch to the member functions - which is ABI dependent. The CPU transformer handles this by only passing the `op_durations` array through the ctx. 

https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/cpu/cpu_runtime_context.hpp#L48

Reply:
I'm not sure we need to take the same approach, but I don't believe what we have is ABI safe yet. We can add a ticket it to address it later if you prefer, but I would say it probably should be done prior to the next version tagging.

Reply:
My understanding is the interface are the following functions:
```
   void start_stopwatch(GPURuntimeContext* ctx, size_t idx);
   void stop_stopwatch(GPURuntimeContext* ctx, size_t idx);
   size_t count_stopwatch(GPURuntimeContext* ctx, size_t idx);
   size_t us_stopwatch(GPURuntimeContext* ctx, size_t idx);
```
only `GPURuntimeContext*` pointer and `size_t` been passed through interface, which should be potable.

Reply:
Got it, misunderstood, thought these were member functions. LGTM.